### ISSUE TITLE
[Bug Report]: Crash On Start Up Caused by Modifying Attack Range

### ISSUE THREAD

**Issue by caniplzjustlogin4godsakesthistakesforev** [chronosacaria/MCDungeonsWeapons#301]
### Describe the bug; What happened?

Have MCDW for 1.20.1 installed with other mods listed in attached debug & logs, Side note this crash does not cause crash report to be generated.

### Expected Behavior

Game loading to menu

### Mod Version

9.0.4

### Minecraft Launcher

Other

### Operating System

Windows

### Additional context

[Log](https://gist.github.com/caniplzjustlogin4godsakesthistakesforev/0bf09a47d29ea1d4302531990e63a91f) & [Debug](https://gist.github.com/caniplzjustlogin4godsakesthistakesforev/ea8541350aae8f89e8f6df3302ca51bf)

**Comment by chronosacaria** [comment]
@caniplzjustlogin4godsakesthistakesforev Believe it or not, I am almost certain that the issue is actually being caused by Valkyrian Skies. Please removing it and try to load. If the issue persists, please post new logs

**Comment by caniplzjustlogin4godsakesthistakesforev** [comment]
Sorry but I'm just going to uploade these directly because I'm lazy: 
[latest.log](https://github.com/user-attachments/files/17176780/latest.log)
[debug.log](https://github.com/user-attachments/files/17176781/debug.log)
(Valkyrian Skies and all related mods are .disabled when running)

**Comment by chronosacaria** [comment]
@caniplzjustlogin4godsakesthistakesforev No, that was fine. I'm genuinely stumped. Can you join the Discord and provide either a CurseForge export or an mrpack file so I can test locally and see what is going on, please?

[https://www.discord.gg/HGQUUSrwt3](https://www.discord.gg/HGQUUSrwt3)
